src/containerDeployment/dockerUtils.ts

```diff
@@ -360,7 +582,7 @@ export async function validateContainerName(containerName: string): Promise<stri
  */
 export async function getDockerPath(executable: string): Promise<string> {
     try {
-        const stdout = (await execCommand(COMMANDS.GET_DOCKER_PATH)).trim();
+        const stdout = await execPowerShellCommand(COMMANDS.GET_DOCKER_PATH());

```

Comment:
Why do some of these need to be powershell, but others are fine to run with whatever the default shell is?